containers/zammad/docker-entrypoint.sh

```diff
@@ -5,6 +5,8 @@ set -e
 : "${AUTOWIZARD_JSON:=''}"
 : "${ELASTICSEARCH_HOST:=zammad-elasticsearch}"
 : "${ELASTICSEARCH_PORT:=9200}"
+: "${ELASTICSEARCH_PROTO:=http}"
```

Comment:
one last thing.... i'd like to rename ELASTICSEARCH_PROTO to ELASTICSEARCH_SCHEMA

Reply:
Very good, Sir. 🖖 It's done, take a 👀?